docs/_docs/configuration/menu-hjson.md

```diff
@@ -362,3 +363,188 @@ newUserApplicationPre: {
     // note that the rest of this menu is omitted for clarity
 }
 ```
+
+## Case Study: Manual Encoding Selection
+
+Enigma½ tries to automatically determine the proper encoding for a client when it connects. Unfortunately, there are cases where the wrong encoding can be selected, resulting in terminal programs that are not supported. If your user base contains users that would like to connect with unsupported clients, one solution is to offer manual encoding selection.
+
+A new system method has been added to support this, @systemMethod:setClientEncoding.
```

Comment:
nit: Maybe not say new here as that would go stale. Perhaps a mention in WHATSNEW though.

Reply:
Done, thanks.